test/unit/middlewares/error-handler.spec.js

```diff
@@ -206,5 +203,34 @@
 					expect(broker.logger.error).toHaveBeenCalledWith(err);
 				});
 		});
+
+		describe('if the error handler resolves the error', () => {
+			beforeEach(() => {
+				broker.errorHandler = jest.fn(() => Promise.resolve(true));
+			});
+
+			it('should return the response from the error handler', () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'should·return·the·response·from·the·error·handler'` with `"should·return·the·response·from·the·error·handler"`](https://app.codacy.com/gh/moleculerjs/moleculer/pullRequest?prid=8141835)
```suggestion
			it("should return the response from the error handler", () => {
```